pulsar-common/src/main/java/org/apache/pulsar/common/protocol/Commands.java

```diff
@@ -180,7 +180,7 @@ public static ByteBuf newConnect(String authMethodName, String authData, String
     private static void setFeatureFlags(FeatureFlags flags) {
         flags.setSupportsAuthRefresh(true);
         flags.setSupportsBrokerEntryMetadata(true);
-        flags.setSupportsPartialProducer(true);
+        // flags.setSupportsPartialProducer(true);
```

Comment:
What is the purpose of leaving this as a comment?

Reply:
it was unintentional! glad you found it before this get merged

reverted now